test/io/testxyzreader.py

```diff
@@ -0,0 +1,116 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2017, the cclib development team
+#
+# This file is part of cclib (http://cclib.github.io) and is distributed under
+# the terms of the BSD 3-Clause License.
+
+"""Unit tests for XYZ reader."""
+
+import os
+import unittest
+
+import numpy as np
+
+import cclib
+
+
+__filedir__ = os.path.dirname(__file__)
+__filepath__ = os.path.realpath(__filedir__)
+__datadir__ = os.path.join(__filepath__, "..", "..")
+
+
+class XYZReaderTest(unittest.TestCase):
+
+    def test_attributes_single(self):
+        """Is an XYZ file with a single geometry read into a ccData properly?
+        """
+        fpath = os.path.join(__datadir__, "test/bridge/uracil.xyz")
+        assert os.path.exists(fpath)
+        xyz = cclib.io.xyzreader.XYZ(fpath)
+        data = xyz.parse()
+
+        attrs = ('natom', 'atomnos', 'atomcoords')
```

Comment:
You can say the same thing on 3 lines explicitly. Also, since you're actually testing these attribute below, I don't think you need to assert their existence.